source/index.html.erb

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta content="IE=edge" http-equiv="X-UA-Compatible">
+  <meta charset="utf-8">
+  <meta content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no" name="viewport">
+
+  <title>EMPEX</title>
+
+  <%= stylesheet_link_tag "landing-all" %>
+  <%= stylesheet_link_tag "//cdn.jsdelivr.net/jquery.slick/1.5.9/slick.css" %>
+  <%= stylesheet_link_tag "//cdn.jsdelivr.net/jquery.slick/1.5.9/slick-theme.css" %>
+
+  <%= favicon_tag "favicon.ico" %>
+
+  <link rel="canonical" href="http://empex.co">
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@empexco" />
+  <meta name="twitter:title" content="EMPEX - An Elixir Movement." />
+  <meta name="twitter:description" content="EMPEX is a sophisticated conference series for the Elixir programming language and ecosystem held in New York City and Los Angeles."
+  />
+  <meta name="twitter:image" content="http://empex.co/assets/images/flag-577c45fc.jpg" />
+
+  <meta property="og:url" content="http://empex.co/" />
+  <meta property="og:title" content="EMPEX - Empire City Elixir Conference 2017" />
+  <meta property="og:description" content="EMPEX, the Empire City Elixir Conference, is a sophisticated conference series for the Elixir programming language and ecosystem held in New York City."
```

Comment:
let's change these OG title and description tags to match the twitter tags above.